Description: Platform-specific fixes
 * Define PATH_MAX, MAX_IOV for Hurd.
Author: Karl Ferdinand Ebert <kfebert@gmail.com>
Author: Romain Francoise <rfrancoise@debian.org>
Bug-Debian: http://bugs.debian.org/609333
Forwarded: not-needed

--- a/compat.h
+++ b/compat.h
@@ -289,6 +303,14 @@ void		 explicit_bzero(void *, size_t);
 int		 getdtablecount(void);
 #endif
 
+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
+
+#ifndef IOV_MAX
+#define IOV_MAX	1024
+#endif
+
 #ifndef HAVE_CLOSEFROM
 /* closefrom.c */
 void		 closefrom(int);
